build-system/tasks/helpers.js

```diff
@@ -602,39 +615,28 @@ async function compileJsWithEsbuild(srcDir, srcFilename, destDir, options) {
 /**
  * Minify the code with Terser. Only used by the ESBuild.
  *
- * @param {string} destDir
- * @param {string} destFilename
- * @param {?Object} options
- * @return {!Promise}
+ * @param {string} code
+ * @param {string} map
+ * @return {!Promise<{code: string, map: *, error?: Error}>}
  */
-async function minifyWithTerser(destDir, destFilename, options) {
-  if (!options.minify) {
-    return;
-  }
-
-  const filename = path.join(destDir, destFilename);
+async function minify(code, map) {
   const terserOptions = {
-    mangle: true,
-    compress: true,
+    mangle: {},
+    compress: {
+      passes: 3,
```

Comment:
Maybe add a comment / link to justify the choice of 3 passes?

Reply:
Sure. Although the answer isn't too satisfying. It was simply experimentation. Minor improvements until `3`, none after that